Fix linter warnings

GoncaloGarcia opened the pull request.

kflansburg commented:
Ah, I see, I fixed here which hasn't merged yet. maybe you can cherry-pick that commit? https://github.com/cloudflare/workers-rs/pull/603

GoncaloGarcia commented:
> Ah, I see, I fixed here which hasn't merged yet. maybe you can cherry-pick that commit? #603

Cherry-picked your commit and deleted my changes since they weren't needed
